testsuite/MDAnalysisTests/analysis/test_hydrogenbonds_analysis.py

```diff
@@ -207,6 +209,140 @@ def test_count_by_time(self, universe):
         assert_array_almost_equal(h.times, ref_times)
         assert_array_equal(counts, ref_counts)
 
+    def test_hydrogen_bond_lifetime(self, universe):
+        hbonds = HydrogenBondAnalysis(universe, **self.kwargs)
```

Comment:
Can this be made a class-level fixture so that it only needs to run once?

Reply:
Or does this create problems because the instance is modified in different ways?

Then it still makes sense as a normal method/function fixture to just reduce code repetition.

Reply:
Added a fixture to replace most of the cases.